lib/network.sh

```diff
@@ -46,7 +46,6 @@ if [[ "${BARE_METAL_PROVISIONER_SUBNET_IPV6_ONLY}" == "true" ]]; then
   export BOOT_MODE="${BOOT_MODE:-UEFI}"
   export BARE_METAL_PROVISIONER_NETWORK="${BARE_METAL_PROVISIONER_NETWORK:-fd2e:6f44:5dd8:b856::/64}"
 else
-  export BOOT_MODE="${BOOT_MODE:-legacy}"
```

Comment:
You may want to keep the default *somewhere*, otherwise you may end up with an empty string. Just move line 46 to common.sh?

Reply:
I have modified this by the time the comment arrived but thanks anyways! I have added a default to lib/common.sh